owner/secure_owner_api_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 BlockCypher
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http//www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package owner_test
+
+import (
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func checkError(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
```

Comment:
Think you forgot to delete some test functions :)
```suggestion
)
```

Reply:
I think you can just checkout this file for now. Would be great to have some real test with a CI.
Like download grin and grin-wallet binaries and test the functions.